packages/tour-kit/src/types.ts

```diff
@@ -14,7 +14,10 @@ export type Step = {
 		// | ...
 	};
 	options?: {
-		classNames?: string | string[];
+		classNames?: {
+			desktop?: string | string[];
+			mobile?: string | string[];
```

Comment:
This could use some documentation as to what constitutes what, since they're the API. Something like the following. These numbers are correct, I checked.

```suggestion
		    /**
		    * `desktop` classes are applied when min-width is larger or equal to 480px.
		    */
		    desktop?: string;
		    /**
		    * `mobile` classes are applied when max-width is smaller or equal to 480px.
		    */
			mobile?: string | string[];
```